packages/manager/core/generator/app/templates/src/hooks/guide/useGuideUtils.tsx

```diff
@@ -1,8 +1,8 @@
-import { useState, useEffect } from 'react';
+import { useContext, useEffect, useState } from 'react';
 import { CountryCode } from '@ovh-ux/manager-config';
-import { useAuthentication } from '@ovh-ux/manager-react-core-application';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 
-const docUrl = 'https://docs.ovh.com/';
+const docUrl = 'https://docs.ovh.com';
```

Comment:
Maybe set this URL in a constant file to provide an exemple of the new folder structure ? with the .constants.ts

Reply:
It cant be usefull if we use different base url  👍 lets the developper create his constant file if necessary